_pkgdown.yml

```diff
@@ -13,7 +13,21 @@ reference:
 - title: Analysis functions
   contents: has_concept("analysis functions")
 - title: Plotting functions
-  contents: has_concept("plotting functions")
+- subtitle: Quick plots
+  contents: starts_with("qplot_")
+- subtitle: Plots
+  contents: starts_with("plot_")
+- subtitle: Preparing data
```

Comment:
I wonder if it may be better to move "Preparing data" ahead of "Plots", since you have to run them in that sequence?

Reply:
sure

Reply:
eh.... but you don't for the "quick plots" right?

Reply:
done

Reply:
anyway, still makes sense to have "prep" stuff first, so I made the change